ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -400,12 +400,19 @@ - (NSDictionary *)powerState
 
 RCT_EXPORT_METHOD(getAvailableLocationProviders:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
 {
+#if !TARGET_OS_TV
```

Comment:
Can you change to if else, please?

Reply:
Seems reasonable, thanks for the review @markhomoki !